src/lib/ggpo/types.h

```diff
@@ -40,16 +40,16 @@ typedef char int8;
 typedef short int16;
 typedef int int32;
 
+
 /*
  * Macros
  */
 #if defined(_DEBUG)
-#define BREAK  DebugBreak();
```

Comment:
Could you call this `DEBUG_BREAK` instead of just `BREAK`?

Reply:
This was actually already included in types.h (I didn't write it), just fixed the formatting. It isn't used anywhere in the codebase, so it should be safe to just remove it.